packages/client/auth-core/src/services/CrossmintAuthService.ts

```diff
@@ -20,6 +20,22 @@ export class CrossmintAuthService extends BaseCrossmintService {
         return {
             jwtToken: result.jwt,
             refreshToken: result.refresh,
+            user: result.user,
         };
     }
+
+    async getUserFromServer(externalUserId: string) {
+        const result = await this.fetchCrossmintAPI(
+            `sdk/auth/user/${externalUserId}`,
```

Comment:
should do go in a versioned route directly?

Reply:
It should go, but making the change will break the current user flow. So, we should duplicate the controller, and I prefer delaying that until needed. As this adds a new endpoint, it's not needed for this iteration

Reply:
okay, though just for my knowledge, what unversioned endpoints are we calling from the sdks right now that would break?